xapian-letor/bin/trainletor.cc

```diff
@@ -0,0 +1,146 @@
+/** @file trainletor.cc
+ * @brief Command line tool to prepare Letor training file and train the model
```

Comment:
I started to review this, but I'm starting to suspect it's been renamed to something else and the old version not removed (though I didn't spot an exact replacement).

@ayshtmr Could you do a pass through and make sure any files which have been renamed (or just removed) have actually been removed from git too (with `git rm`).  That'll save both of us time and effort - there's no point me giving feedback on dead code which you then have to wade through and try to work out if it applies to a relocated version somewhere.